Formula/open-mpi.rb

```diff
@@ -44,12 +44,14 @@ def install
       oshmem/tools/oshmem_info/param.c
     ]
 
-    inreplace inreplace_files, "OMPI_CXX_ABSOLUTE", "\"#{ENV.cxx}\""
+    cxx = OS.mac? ? "clang" : "g++"
```

Comment:
```suggestion
    cxx = OS.linux? ? "g++" : ENV.cxx
```
`clang` is wrong here.